src/tests/conformance/query/peer_shard_info.rs

```diff
@@ -13,6 +13,7 @@ async fn node_should_query_for_shard_info_after_handshake() {
         |m: &BinaryMessage| matches!(&m.payload, Payload::TmGetPeerShardInfoV2(..));
     perform_response_test(Default::default(), &response_check).await;
 }
+
```

Comment:
what's this :detective:?

Reply:
I added an empty line between two tests (function blocks) since that is how it was done for other tests in other files.